pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarClientImpl.java

```diff
@@ -710,14 +713,30 @@ public void close() throws PulsarClientException {
         final CompletableFuture<Void> closeFuture = new CompletableFuture<>();
         List<CompletableFuture<Void>> futures = new ArrayList<>();
 
-        producers.forEach(p -> futures.add(p.closeAsync()));
-        consumers.forEach(c -> futures.add(c.closeAsync()));
+        producers.forEach(p -> futures.add(p.closeAsync().handle((__, t) -> {
+            log.error("Error closing producer {}", p, t);
```

Comment:
this is an error only if "t" is not null

Reply:
thanks, I'll fix it.

Reply:
@eolivelli fixed now. PTAL